### PR TITLE
Feat/grafana dashboards with charm

### PR DESCRIPTION
**Pull request by Guillaumebeuzeboc**
Write grafana dashboards in a folder so the charm can export it.

### CODE REVIEW

File: .github/workflows/rock-release-pr.yaml
Comment (artivis) [inline .github/workflows/rock-release-pr.yaml:1]: This workflow seems to be dealing with both PRs and releases.
Shouldn't the two concerns be separated?
It runs only on PR to the main branch and upload the resulting image to ghrc?

File: cos_registration_server/api/dashboards.py
Comment (artivis) [inline cos_registration_server/api/dashboards.py:1]: Il this deals only with grafana dashboard, should we use a more specific naming (e.g. `grafana_dashboard`)?

Comment (artivis) [inline cos_registration_server/api/dashboards.py:1]: Add some docstrings please.

Comment (Guillaumebeuzeboc) [reply cos_registration_server/api/dashboards.py:1]: All the docustrings are waiting in the next PR: https://github.com/ubuntu-robotics/cos-registration-server/pull/12

Comment (Guillaumebeuzeboc) [reply cos_registration_server/api/dashboards.py:1]: yes indeed, we won't need that for foxglove layouts.

**Review by artivis — APPROVED** [review]

File: .github/workflows/rock-release-pr.yaml
Comment (Guillaumebeuzeboc) [reply .github/workflows/rock-release-pr.yaml:1]: I will add a tag depending on the branch name in another PR.

Comment (Guillaumebeuzeboc) [reply .github/workflows/rock-release-pr.yaml:1]: I can also add a cron workflow that cleans tags from non-existant branch.

**Pull request merged**